packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/backups.html

```diff
@@ -12,21 +12,26 @@ <h2 data-translate="pci_databases_backup"></h2>
     <oui-tile
         heading="{{:: 'pci_databases_backup_informations_title' | translate}}"
     >
-        <div class="oui-table">
-            <div>
-                <div class="oui-table__row">
-                    <div class="oui-table__cell">
-                        <div class="oui-description">
-                            <div
-                                data-translate="pci_databases_backup_informations_retention"
-                            ></div>
-                            <div>
-                                {{:: $ctrl.backupRetentionTime.humanize()}}
-                            </div>
-                        </div>
-                    </div>
+        <div class="row px-3">
+            <div class="col border-top border-bottom border-light p-2">
+                <div
+                    data-translate="pci_databases_backup_informations_retention"
+                ></div>
+                <div>
+                    {{:: $ctrl.backupRetentionTime.humanize()}}
```

Comment:
**suggestion:** Prefer using `ng-bind` instead of `{{ }}` 
```suggestion
                <div data-ng-bind=":: $ctrl.backupRetentionTime.humanize()">
```